src/main/java/the_warlord/actions/TripleSlashAction.java

```diff
@@ -0,0 +1,77 @@
+package the_warlord.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import the_warlord.cards.warlord.TripleSlashCopy;
+
+import java.util.ArrayList;
+
+public class TripleSlashAction extends AbstractGameAction {
+    private AbstractMonster targetMonster;
+    private boolean upgraded;
+    private ArrayList<AbstractCard> list = new ArrayList<>();
+    AbstractPlayer p;
+
+    public TripleSlashAction(boolean upgraded, AbstractMonster targetMonster) {
```

Comment:
This custom action is not necessary, since we already have the CardMetaUtils::playCardAdditionalTime method. See the comment on TripleSlash::use for how to use it!